### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by RDIL**
Major highlights:
* Updated PrismJS
* Updated Babel from v7 to v7.8
* Updated React
* Updated TSLint
* Updated image shrinkers

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -2,11 +2,10 @@ ports:
 - port: 8000
 tasks:
 - init: yarn && yarn build
-  command: yarn dev --host 0.0.0.0
+  command: gatsby telemetry --disable && yarn dev --host 0.0.0.0
```
Comment (nisarhassan12) [inline .gitpod.yml]: Invoking the `gatsby` cli won't work here i think you have to first install it globally

Comment (nisarhassan12) [reply .gitpod.yml]: I would recommend not installing it globally instead invoking it from a package.json script. Currently `yarn dev --host 0.0.0.0` is not working 
image

Comment (RDIL) [reply .gitpod.yml]: Fixed

**Comment by nisarhassan12** [comment]
@RDIL Thanks!  LGTM :slightly_smiling_face:

**Comment by RDIL** [comment]
You guys should consider installing [Dependabot](https://dependabot.com)

**Comment by nisarhassan12** [comment]
> You guys should consider installing [Dependabot](https://dependabot.com)

Thanks @RDIL We use to have it installed but we had some issues so we removed it. 
@jankeromnes  What do you think would it make sense to re-add it again?

**Comment by svenefftinge** [comment]
I found it too noisy and prefer regularly updating the dependencies.

**Comment by svenefftinge** [comment]
@nisarhassan12 can you rebase and merge?

**Comment by jankeromnes** [comment]
Thanks for the suggestion @RDIL! I really like Dependabot (and Greenkeeper before that), but for this repo it was constantly spamming us with small Pull Requests (due to frequent monorepo updates I think), which was a bit challenging to keep up with. Also, we've then merged upgrades that accidentally broke smaller parts of the website, because we didn't have time to sufficiently test each PR. So we gave up on it for now (better a slightly out-dated website that works well than a very up-to-date website that constantly breaks).

**Comment by nisarhassan12** [comment]
> @nisarhassan12 can you rebase and merge?

Done!

**Pull request merged**